config/locales/client.en.yml

```diff
@@ -26,3 +26,10 @@ en:
       change_settings: "Change Settings"
       no_sift_info: "No Sift info available"
       no_sift_tokenized: "No Sift tokenized breakdown available"
+
+    topic:
+      deleted: "[Topic Deleted]"
```

Comment:
Didn't we add this into core for the reviewables?

Reply:
Wasn't merged yet. Will remove